ci/base.yml

```diff
@@ -29,5 +29,8 @@ local_conf_header:
     INHERIT += "buildstats buildstats-summary"
     INHERIT += "buildhistory"
     INHERIT += "rm_work"
+  mirror: |
```

Comment:
it's a good change for build speed, no doubt. However it also means that we will use most of Yocto packages as 'binaries' without rebuilding them. Should we have any concerns about the security implications here? An alternative approach would be to build our own internal mirror, populated from our builds.

Reply:
> However it also means that we will use most of Yocto packages as 'binaries' without rebuilding them. Should we have any concerns about the security implications here? 

This is only true if we use the same configuration and sources that is used upstream and if this is true the binaries produced are the same because yocto is reproducible bit by bit.
In terms of security, the consequences will be the same because if we compile our software using the same sources that exist upstream in yocto, we will obtain the same binary that yocto produces for the same reproducible reasons.

we have already this task of our own mirror planed https://github.com/quic-yocto/meta-qcom-hwe/issues/13